templates/tailwind/block/feature-section/configuration/feature-section-schema.json

```diff
@@ -0,0 +1,117 @@
+{
+  "$schema": "https://schema.croct.com/json/v1/template-content-schema.json",
+  "type": "structure",
+  "attributes": {
+    "image": {
+      "type": {
+        "id": "@croct/file",
+        "type": "reference",
+        "properties": {}
+      },
+      "label": "Image",
+      "private": false,
+      "optional": false,
+      "position": 4,
+      "description": "The section image."
+    },
+    "title": {
+      "type": {
+        "type": "text",
+        "minimumLength": 1
+      },
+      "label": "Title",
+      "private": false,
+      "optional": false,
+      "position": 1,
+      "description": "The section title."
+    },
+    "features": {
+      "type": {
+        "type": "list",
```

Comment:
We forgot to add a constraint for non-empty lists. I would also add a maximum of 3 elements.

<img width="1354" alt="image" src="https://github.com/user-attachments/assets/519de7da-e91f-460c-82f2-307aee78c119" />